stdlib/testing/testdata/filter_nulls.flux

```diff
@@ -0,0 +1,57 @@
+package testdata_test
```

Comment:
Does it make sense to rename the files for these tests to `_exists` instead of `_nulls` as the function under test is the `exists` function?

Reply:
yeah, good point. will change.